### PR TITLE
Get whole error node, not just first. Check for 'Errors' instead of just 'errors'

### PR DESCRIPTION
**Pull request by oochelz**

### CODE REVIEW

File: lib/cb/utils/validator.rb [hunk 1]
```diff
@@ -64,6 +63,16 @@ def try_parse_xml(body)
...
       end
+
+      def error_message(processed_response)
+        if processed_response['errors']
+          processed_response['errors']
+        elsif processed_response['Errors']
+          processed_response['Errors']
+        else
+          ''
+        end
```
Comment (ausmith) [inline lib/cb/utils/validator.rb]: Using `.fetch` I think we could have done this with 1 line, but it really doesn't matter :)

Comment (oochelz) [inline lib/cb/utils/validator.rb]: How would `fetch` help? I'm not terribly familiar... But the documentation makes no mention of case insensitivity.

Comment (oochelz) [inline lib/cb/utils/validator.rb]: @ausmith Still curious how it would work with case differences. Do you know?

Comment (ausmith) [inline lib/cb/utils/validator.rb]: Sorry about not seeing this comment before. The definition of `#fetch` is: `fetch(key [, default])`

So we can get a little fancy with it and use the default to do the alternative lookup or just return empty string:

`processed_response.fetch('errors', processed_response('Errors', ''))`

Comment (oochelz) [inline lib/cb/utils/validator.rb]: Oo, I like that.

**Pull request merged**